src/Logger.spec.ts

```diff
@@ -18,7 +18,7 @@ describe('Logger', () => {
     });
     it('should queue if no factory is installed', (done) => {
       Logger.getLogger(__filename);
-      log?.debug('aloha');
+      log.debug('aloha');
```

Comment:
This test should probably test the queued events?

If the changed logic in the queued messages from comment https://github.com/surikaterna/slf/pull/9#discussion_r942066811 can be proven to work here, we can be more confident that the change does not break something that works now.